.travis.yml

```
@@ -7,8 +7,8 @@ env:
     - SEGFAULT_SIGNALS=all
     - SALT_STATE=scap.scan
   matrix:
-    - OS_VERSION=7 SALT_PILLARROOT=$TRAVIS_BUILD_DIR/tests/pillar/test-scc
-    - OS_VERSION=7 SALT_PILLARROOT=$TRAVIS_BUILD_DIR/tests/pillar/test-oscap
+    - OS_VERSION=7 SALT_PILLARROOT=$TRAVIS_BUILD_DIR/tests/pillar/test-scc SALT_VERSION=3004
```

Should be able to move `SALT_VERSION` to `global:` to avoid duplicating the usage in the `matrix:`.

```suggestion
    - OS_VERSION=7 SALT_PILLARROOT=$TRAVIS_BUILD_DIR/tests/pillar/test-scc
```